mne_bids_pipeline/_report.py

```diff
@@ -69,14 +69,13 @@ def _open_report(
             yield report
         finally:
             try:
-                msg = "Adding config and sys info to report"
-                logger.info(**gen_log_kwargs(message=msg))
```

Comment:
Moved this later in the file where we can make sure we emit it exactly once per `mne_bids_pipeline` call (other calls should be fast and it's not helpful to have dozens of these messages in the logs)